src/api.js

```diff
@@ -1,28 +1,34 @@
 // import {renderInfoCard, renderWaterInfo, renderActivityInfo, renderSleepInfo, createInitialDashboard} from "./scripts";
 
-const getUserData = () => {
-  return fetch("http://localhost:3001/api/v1/users")
-    .then(response => response.json())
-    .then((data) => {return data})
-};
-
-const getSleepData = () => {
-  return fetch("http://localhost:3001/api/v1/sleep")
-    .then(response => response.json())
-    .then((data) => {return data})
-};
+// const getUserData = () => {
+//   return fetch("http://localhost:3001/api/v1/users")
+//     .then(response => response.json())
+//     .then((data) => {return data})
+// };
+//
+// const getSleepData = () => {
+//   return fetch("http://localhost:3001/api/v1/sleep")
+//     .then(response => response.json())
+//     .then((data) => {return data})
+// };
+//
+// const getActivityData = () => {
+//   return fetch("http://localhost:3001/api/v1/activity")
+//     .then(response => response.json())
+//     .then((data) => {return data})
+// };
+//
+// const getHydrationData = () => {
+//   return fetch("http://localhost:3001/api/v1/hydration")
+//     .then(response => response.json())
+//     .then((data) => {return data})
+// };
 
-const getActivityData = () => {
-  return fetch("http://localhost:3001/api/v1/activity")
-    .then(response => response.json())
-    .then((data) => {return data})
-};
+const fetchData = (param) => {
+    return fetch(`https://pacific-badlands-43237.herokuapp.com/api/v1/${param}`)
```

Comment:
Accidentally, did not change the link back to the new one... easy fix next time we are working.